client/src/pages/private/SiteViewDetails.js

```diff
@@ -50,15 +55,28 @@ export default ({ match }) => {
   };
 
   const fetchDetails = useCallback(async () => {
-    const response = await fetch(`${API_URL}/api/v1/employer-sites/${id}`, {
-      headers: {
-        Authorization: `Bearer ${store.get('TOKEN')}`,
-      },
-      method: 'GET',
-    });
-
+    setIsLoading(true);
+    const response = await fetchSite(id);
     if (response.ok) {
-      setSite(await response.json());
+      const site = await response.json();
+      let phases = [];
+      if (featureFlag(flagKeys.FEATURE_PHASE_ALLOCATION)) {
+        phases = await fetchSitePhases(site.id);
+      }
+      const participants = await fetchSiteParticipants(columnIDs, site.siteId);
+      setIsLoading(false);
+      return setSite({
+        ...site,
+        hiredParticipants: participants.hiredParticipants,
+        withdrawnParticipants: participants.withdrawnParticipants,
+        phases,
+      });
+    } else {
+      openToast({
+        status: ToastStatus.Error,
+        message: response.error || response.statusText || 'Server error',
+      });
+      return setSite({ hiredParticipants: [], withdrawnParticipants: [], phases: [] });
     }
   }, [id, setSite]);
```

Comment:
Pretty sure `openToast` should be in this array.

Reply:
updated! Thanks for pointing that out

Reply:
Great! It might be worth ensuring you have eslint integration enabled in your editor - it has caught details like this many times for me.